packages/ng/user/select/input/user-select-input.component.ts

```diff
@@ -61,6 +61,7 @@ export class LuUserSelectInputComponent<U extends import('../../user.model').ILu
 	@Input() appInstanceId: number | string;
 	@Input() operations: number[];
 	@Input() enableFormerEmployees = false;
+	@Input() enablePrincipal = true;
```

Comment:
A boolean that defaults to `true` is always weird, why not name it `disablePrincipal`?

Reply:
I wasn't very comfortable to write `*ngIf="!disablePrincipal"` within the template, looks like a negation of a negation in my mind 
whatever, I'll do the refactoring 😄